README.md

```diff
@@ -31,6 +31,21 @@ git submodule update --remote
 
 and commit the result.
 
+
+Developing
+----------
+
+If you add a shellscript consider using [bats](https://github.com/bats-core/bats-core) to add tests for it.
```

Comment:
I think we can keep the decision of which testing framework adoption for shell script local to this project, and use it elsewhere if it proves very successful. Which means this is easy to approve.